instrument-functions.c

```diff
@@ -158,16 +158,13 @@ static void print_debug(void *this_fn, void *call_site, action_type action)
 
 	if (instrument_global) {
 		symbol_info syminfo;
-		int found;
-
-		i = 0;
-		found = 0;
-		while (i < symcount && !found) {
+		int found = 0;
+		for (i = 0; i < symcount; i++) {
```

Comment:
Why this change?

Reply:
It looks more intuitive. But if I must change it back I will.

Reply:
@fxlb Changed back!